hack/lib/util.sh

```diff
@@ -437,20 +437,24 @@ kube::util::ensure_clean_working_dir() {
 
 # Ensure that the given godep version is installed and in the path.  Almost
 # nobody should use any version but the default.
+#
+# Sets:
+#  GODEP: The name of the godep binary
+#
 kube::util::ensure_godep_version() {
   GODEP_VERSION=${1:-"v79"} # this version is known to work
 
   if [[ "$(godep version 2>/dev/null)" == *"godep ${GODEP_VERSION}"* ]]; then
+    GODEP="$(which godep)"
     return
   fi
 
   kube::log::status "Installing godep version ${GODEP_VERSION}"
-  go install ./vendor/github.com/tools/godep/
-  GP="$(echo $GOPATH | cut -f1 -d:)"
-  hash -r # force bash to clear PATH cache
-  PATH="${GP}/bin:${PATH}"
+  mkdir -p "${KUBE_ROOT}/_tmp/bin"
+  GOBIN="${KUBE_ROOT}/_tmp/bin" go install ./vendor/github.com/tools/godep/
```

Comment:
should this be setting `GOPATH` instead? i.e. will `go install` here still pollute the user's `$GOPATH/pkg`?